library/std/src/sync/condvar.rs

```diff
@@ -305,7 +305,11 @@ impl Condvar {
     /// }
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
-    #[deprecated(since = "1.6.0", note = "replaced by `std::sync::Condvar::wait_timeout`")]
+    #[deprecated(
+        since = "1.6.0",
+        note = "replaced by `std::sync::Condvar::wait_timeout`",
+        suggestion = "wait_timeout"
```

Comment:
This doesn't seem correct, as the suggestion lint message is `Applicability::MachineApplicable`, and `wait_timeout` accepts a different type to `wait_timeout_ms`.